### PR TITLE
Baseframe form id

### PR DESCRIPTION
**Pull request by vidya-ram**
1. If client has passed a `formid`, prefix it with 'form-' and use it as id attribute for the form.
2. Incase the client doesn't pass a `formid`, generate one.

### CODE REVIEW

File: baseframe/forms/auto.py [hunk 1]
```diff
@@ -18,9 +18,12 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
+def render_form(form, title, message='', formid='', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
```
Comment (jace) [inline baseframe/forms/auto.py]: Use `None` instead of `''` for the default value.

File: baseframe/forms/auto.py [hunk 2]
```diff
@@ -18,9 +18,12 @@ class ConfirmDeleteForm(Form):
...
     multipart = False
-    ref_id = 'form-' + buid()
+    if not formid:
+        ref_id = 'form-' + buid()
+    else:
+        ref_id = 'form-' + formid
```
Comment (jace) [inline baseframe/forms/auto.py]: This could be `ref_id = 'form-' + (formid or buid())`

File: baseframe/forms/auto.py (hunk 1 above)
Comment (vidya-ram) [reply baseframe/forms/auto.py]: Fixed

File: baseframe/forms/auto.py (hunk 2 above)
Comment (vidya-ram) [reply baseframe/forms/auto.py]: Fixed

**Pull request merged**